packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -34,6 +39,51 @@ final dialogTextFieldDecoration = InputDecoration(
   ),
 );
 
+/// A dialog, that reports unexpected error and allows to copy details and create issue.
+class UnexpectedErrorDialog extends StatelessWidget {
```

Comment:
Can you add a screenshot of the updated dialog to the PR comments? :-)

Reply:
<img width="1046" alt="Screenshot 2023-03-28 at 10 32 47 AM" src="https://user-images.githubusercontent.com/12115586/228321490-e0e46adb-4be3-4479-905c-52b6314faa92.png">

Reply:
It is scrollable when text is too large.